alphagov/pay-connector
# PP 11594 Allow enabling Apple Pay and Google Pay for Sandbox gateway accounts

## WHAT YOU DID
Allow enabling Apple Pay and Google Pay for Sandbox gateway accounts

[https://payments-platform.atlassian.net/browse/PP-11594](https://payments-platform.atlassian.net/browse/PP-11594)

Allow making a PATCH request to /v1/api/accounts/{accountId} with path allow_apple_pay and allow_google_pay for an account with any payment provider, rather than restricting by payment provider.

Remove throwIfNotDigitalWalletSupportedGateway method and associated unit test as payment  provider restriction lifted

## How to test

- Ensure current PACT tests run successfully and create  new tests if required.
- Manually test toggling Apple Pay and Google Pay on and off in the 'Settings / Digital wallet' section of the Pay Admin tool for sandbox accounts
